jsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es2017",
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./*"]
+    }
+  },
+  "exclude": ["node_modules"]
+}
```

Comment:
Esse arquivo é algo específico do Vscode? Se for, faz sentido ele estar no projeto?

Reply:
Esse arquivo que configura os alias